package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@websdk/nap",
-  "version": "0.9.8",
+  "version": "0.10.0",
```

Comment:
No, using `npm version`. I'm going with the no-one-really-cares-about-semver-before-1.0 approach, so bumping `minor` for the breaking changes and `patch` for non-breaking. Once we hit `1.0.0` though we should take better care.